beacon-chain/core/helpers/committee_test.go

```diff
@@ -507,3 +612,42 @@ func TestCommitteeAssignment_CantFindValidator(t *testing.T) {
 		t.Errorf("Unexpected error: %v", err)
 	}
 }
+
+func BenchmarkCommitteeAssignment(b *testing.B) {
```

Comment:
The benchmark doesn't look right either, we never save the cache in `CommitteeAssignment`, so both benchmarks are benchmarking the same thing. I suspect the performance gain that you get is from easier retrieval of validator with index 105 compared to 800. If you switch the indexes around you should see the opposite happening for each benchmark

Reply:
This was also affected by my later changes, I've made sure it works now:
```
BenchmarkCommitteeAssignment/Uncached-4         	     500	   4661218 ns/op	 1622607 B/op	     229 allocs/op
BenchmarkCommitteeAssignment/Cached-4           	    5000	        99.5 ns/op	       0 B/op	       0 allocs/op
```